contracts/Strategy.sol

```diff
@@ -752,28 +822,47 @@ contract Strategy is BaseStrategy {
         return investmentToken.balanceOf(address(this));
     }
 
-    function balanceOfDebt() public view returns (uint256) {
-        return MakerDaiDelegateLib.debtForCdp(cdpId, ilk);
+    function balanceOfInvestmentTokenInBentoBox() internal view returns (uint256) {
+        return bentoBox.toAmount(investmentToken, bentoBox.balanceOf(investmentToken, address(this)), false);
```

Comment:
@newmickymousse understand we want to save gas but it is helpful to leave balances public if possible.

Reply:
I agree, but the contract is large in size. If I change the visibility, then we exceed the max allowed size.

Reply:
OK, let's make it a nice to have if there is some space left at the end :P